subsys/net/lib/nrf_cloud/src/nrf_cloud_fota_poll.c

```diff
@@ -333,9 +334,8 @@ static int start_download(void)
 	}
 
 	LOG_INF("Starting FOTA download of %s/%s", job.host, job.path);
-	int err = fota_download_start_with_image_type(job.host, job.path,
```

Comment:
with your latest update, this all changes can be reverted from this file?
unless the plan is to deprecate `fota_download_start_with_image_type`?

Reply:
Yeah, you're right, reverted as there is no plan to deprecate